hamp_radar/iquick.py

```diff
@@ -12,6 +12,28 @@
 from decoders import decode_srvi, decode_moment, decode_iq, decode_time
 
 
+def get_tag_size(data):
+    """
+    Extracts the tag (also known as signature) and the pointer for size
+    from 'data' assuming layout as in Meteorological Ka-Band Cloud Radar
+    MIRA35 Manual, section 2.3.1 'Definition of chunk common structure'
+    where the tag is the first 4 bytes of data and the pointer is the
+    next 4 bytes.
+
+    Args:
+        data (bytes): The data assumed to conform with Meteorological Ka-Band
+                      Cloud Radar MIRA35 Manual, section 2.3.1 'Definition of
+                      chunk common structure'
+
+    Returns:
+        Tuple[bytes, int]: The tag and the pointer for size.
```

Comment:
Would you add a note that None is used to signal that no tag could be found?

Reply:
done whoops :)